tests/sentry/api/endpoints/test_user_notification_details.py

```diff
@@ -29,22 +34,29 @@ def test_superuser(self):
         self.get_valid_response(user.id)
 
     def test_returns_correct_defaults(self):
+        """
+        In this test we add existing per-project and per-organization
+        Notification settings in order to test that defaults are correct.
+        """
         user = self.create_user(email="a@example.com")
         org = self.create_organization(name="Org Name", owner=user)
 
-        # Adding existing UserOptions for a project or org to test that defaults are correct
         # default is 3
-        UserOption.objects.create(
-            user=user, project=None, organization=org, key="deploy-emails", value=1
```

Comment:
I have the other PR opened side by side but I'm having a hard time finding what 1 means, can you point me to it?

Reply:
if value is a `UserOptionValue` I'm still not sure what 1 means given the values are 2 for all/yes, 3 for committed deploys only, and 4 for no/off. How did you decide on never?

Reply:
This is a bug. `value=1` should never be set for a "deploy" notification.

https://github.com/getsentry/sentry/blob/6d9bd3f587839f0a4e71dd9da8d3683b96514ea1/src/sentry/models/useroption.py#L14-L17

The test doesn't care what values we put in for per-organization settings as long they aren't the the same as the defaults. The test is just checking that the defaults don't get changed.